pkg/wallet/wallet.go

```diff
@@ -63,12 +62,58 @@ func (w *BaseWallet) DecryptMessage(encMessage []byte, toVerKey string) ([]byte,
 
 // PackMessage Pack a message for one or more recipients.
 func (w *BaseWallet) PackMessage(envelope *Envelope) ([]byte, error) {
-	return nil, fmt.Errorf("not implemented")
+	if envelope == nil {
+		return nil, errors.New("envelope argument is nil")
+	}
+	// get keypair from db
+	senderKeyPair, err := w.getKey(envelope.FromVerKey)
+	if err != nil {
+		return nil, fmt.Errorf("failed from getKey: %w", err)
+	}
+
+	var recipients []*[crypto.KeySize]byte
+	for _, verKey := range envelope.ToVerKeys {
+		// decode base58 ver key
+		verKeyBytes := base58.Decode(verKey)
```

Comment:
I'm still hesitant about this logic.

It is possible to have different key schemes in an interop situation. I wouldn't expect guarantee that each recipient is using the same key types. Yet another TODO...

Reply:
ok

Reply:
I assume #272 applies here too.

Reply:
I added TODO